apps/framework-docs/src/pages/moose/apis.mdx

```diff
@@ -38,6 +38,13 @@ The APIs module provides standalone HTTP endpoints for data ingestion and analyt
   ctaLabel="Learn More →"
   badge={{ variant: "moose", text: "Trigger" }}
 />
+<CTACard
+  title="Bring Your Own API Framework"
+  description="Use Express, Koa, Fastify, or FastAPI with MooseStack for advanced HTTP features and custom middleware"
+  ctaLink="/moose/app-frameworks"
+  ctaLabel="Learn More →"
+  badge={{ variant: "moose", text: "Custom" }}
+/>
```

Comment:
Also add a callout to the overview section on the APIs page (https://framework-docs-git-dseleno-choredoc-byof.preview.boreal.cloud/moose/apis/analytics-api#overview) - something like: "If you'd prefer to use your own API framework of choice, instead of Moose APIs, see the [Bring Your Own API Framework documentation](https://framework-docs-git-dseleno-choredoc-byof.preview.boreal.cloud/moose/app-frameworks) for comprehensive examples and patterns using frameworks such as Express, Koa, Fastify, or FastAPI."